src/ducks/transactions/TransactionModal.jsx

```diff
@@ -269,11 +269,12 @@ const TransactionModalInfoContent = withTransaction(props => {
         </Bd>
         {shouldShowRestoreApplicationDateIcon ? (
           <Img>
-            <Icon
+            <span
               onClick={handleResetApplicationDate}
-              color="var(--slateGrey)"
-              icon="restore"
-            />
+              className="u-expanded-click-area"
+            >
```

Comment:
This should be backported to UI. We should have an `expanded-click-area` on all `Icons` where we have an `onClick` prop

Reply:
https://github.com/cozy/cozy-ui/issues/1263 Issue created :)